oaknational/Oak-Web-Application
# fix: mathjax bug add client provider

## Description

- Moves mathjax context to provider, explicitly states client function in next
- Moves provider to wrap just quizzes rather than page

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2043--oak-web-application.netlify.thenational.academy/teachers/programmes/maths-secondary-ks4-higher-l/units/adding-surds-a57d/lessons/subtract-two-surds-6njkac#starter-quiz
2. Click on the surds lessons - mathjax should still render with no errors


## Checklist
